### PR TITLE
Create get-generation-random-numbers.yml

### PR DESCRIPTION
**Pull request by johnk3r**
https://github.com/fireeye/capa-rules/issues/21

### CODE REVIEW

File: data-manipulation/prng/get-generation-random-numbers.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: get generation random numbers
```
Comment (mr-tz) [inline data-manipulation/prng/get-generation-random-numbers.yml:3]: I suggest `generate random numbers via WinAPI`

File: data-manipulation/prng/get-generation-random-numbers.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    examples:
+      - ba947eb07d8c823949316a97364d060f:0x1400031E0
+      - 3ca359f5085bb96a7950d4735b089ffe:0x403A80
+      - e59ffeaf7acb0c326e452fa30bb71a36:0x40403E
+      - 1195d0d18be9362fb8dd9e1738404c9d:0x404E90
+    mbc:
+      - Cryptography::Generate Pseudo-random Sequence [C0021]
```
Comment (mr-tz) [inline data-manipulation/prng/get-generation-random-numbers.yml:17]: see https://github.com/MBCProject/mbc-markdown/blob/master/micro-behaviors/cryptography/gen-random.md
`Cryptography::Generate Pseudo-random Sequence::Use API [C0021.003]`

File: data-manipulation/prng/get-generation-random-numbers.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: get generation random numbers
+    namespace: data-manipulation/prng
+    author:
+      - michael.hunhoff@fireeye.com
+      - johnk3r
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
```
Comment (mr-tz) [inline data-manipulation/prng/get-generation-random-numbers.yml:10]: Random numbers can be used for a variety of tasks, so I suggest to remove this here.

**Review by mr-tz — CHANGES_REQUESTED** [review]
Thanks, I have a couple of comments on the meta data.

File: data-manipulation/prng/get-generation-random-numbers.yml (hunk 2 above)
Comment (johnk3r) [reply data-manipulation/prng/get-generation-random-numbers.yml]: Nice. I'm still getting acquainted with MBC...

File: data-manipulation/prng/get-generation-random-numbers.yml (hunk 3 above)
Comment (johnk3r) [reply data-manipulation/prng/get-generation-random-numbers.yml]: Got it.

**Comment by johnk3r** [comment]
Thanks for the feedback. Adjustments made.

**Review by mr-tz — APPROVED** [review]
thanks, lgtm

**Comment by mr-tz** [comment]
Need to rename the file to make the linter happy.

**Pull request merged**